tests/README_INTEGRATION_TESTS.md

```diff
@@ -0,0 +1,159 @@
+# Graphical Editor Integration Tests
```

Comment:
Remove this README and instead add relevant information to the header of each test file